tsconfig.json

```diff
@@ -3,6 +3,7 @@
     "strict": true,
     "target": "ESNext",
     "module": "NodeNext",
-    "moduleResolution": "NodeNext"
+    "moduleResolution": "NodeNext",
+    "checkJs": true
```

Comment:
checkJs is needed to replace the jsconfig.json.  maxNodeModuleJsDepth is also different, but I think the jsconfig default is weirder, so I'm going to leave that alone unless you have a different opinion.